content/docs/schedules.md

```diff
@@ -103,9 +105,17 @@ You can add multiple actor and task runs to a schedule with a single POST reques
 
 For more information, see the [schedules section](https://docs.apify.com/api/v2#/reference/schedules/schedule-object/get-schedule) in the API documentation.
 
+## Schedule setup
+
+The schedule setup tool uses [cron expressions](https://en.wikipedia.org/wiki/Cron#CRON_expression), which we parse using the [cron-parser](https://www.npmjs.com/package/cron-parser) package, to specify run times. If you're familiar with how to use them and need a specific run schedule, you can dive right in. If not, don't worry - the setup tool has a visual custom schedule builder that provides the same level of control cron expressions.
+
+![Schedule setup tool]({{@asset images/schedule-setup.webp}})
+
+You can find more information and examples of cron expressions on [crontab.guru](http://crontab.guru/). For additional and non-standard characters, see [this](https://en.wikipedia.org/wiki/Cron#CRON_expression) Wikipedia article.
```

Comment:
I'd mention the "live feedback" that gives you 5 (or so) next runs, so you can experiment with it.